CHANGELOG.md

```diff
@@ -5,31 +5,115 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-### Added
-- Adding slide for smart account upgrade to home page carousel ([#32048](https://github.com/MetaMask/metamask-extension/pull/32048))
```

Comment:
It looks like some of these entries were deleted by accident. Some of these changes are still unreleased as of v12.17.0 (e.g. the first two, I didn't check them all)